Dockerfile

```diff
@@ -42,10 +43,16 @@ RUN for i in /tmp/apt/*.key; do apt-key add $i; done && \
 RUN /usr/bin/pip --no-cache-dir install awscli awsrequests testinfra && \
     /usr/bin/pip3 --no-cache-dir install awscli awsrequests testinfra
 
+# updated golang
+RUN wget https://dl.google.com/go/go1.12.7.linux-amd64.tar.gz
```

Comment:
https://golang.org/dl/ provides checksums to validate the downloaded files - could you add a step to validate the download?  Check out the `gcr_cred_helper_sha256` for an example where the provider just gives you a single hardcoded checksum